pkg/logging/logger.go

```diff
@@ -102,7 +87,7 @@ func (l Logger) Error(err error, message string, kvpair ...interface{}) {
 	}); wrapped {
 		err = wErr.Unwrap()
 	}
-	if err == nil || k8serr.IsConflict(err) {
```

Comment:
Filtering for _Conflict_ more appropriate for the controller `Reconcile()`.
No hardship.